README.md

```diff
@@ -1,21 +1,74 @@
 # MXquickconnect
 
-MXquickconnect is a sample application which anyone can clone and use their MX client credentials to interface with common MX use cases. It is meant to help a developer see a sample of what MX has to offer while at the same time providing some frontend and backend code which can be extracted into any application.
+MXquickconnect is a sample application which anyone can clone and use their MX
+client credentials to interface with common MX use cases. It is meant to help
+a developer see a sample of what MX has to offer while at the same time
+providing some frontend and backend code which can be extracted into any
+application.
 
-First, create a .env file at the same level as the `.env.example`. Add your `CLIENT_ID` and `API_KEY`. You can find these values in your [MX Client Dashboard](https://dashboard.mx.com)
+## Table of contents
 
-Open two terminals, one for the [frontend](#frontend) and one for the backend. You can choose to run **one** of the following backend implementations:
-* [Ruby](#backend-ruby)
-* [Node.js](#backend-nodejs)
-* [Python3](#backend-python)
- ---
-#### Backend (Python)
+<!-- toc -->
+- [1. Clone the repository](#1-clone-the-repository)
+- [2. Set up development environment](#2-set-up-development-environment)
+- [3. Run the MXquickconnect](#3-run-the-mxquickconnect)
+  - [Run without Docker](#run-without-docker)
+    - [1. Running the backend](#1-running-the-backend)
+      * [Ruby](#backend-ruby)
+      * [Node.js](#backend-nodejs)
+      * [Python3](#backend-python)
+      * [DotNet-Core](#backend-aspnet)
+    - [2. Running the frontend](#2-running-the-frontend)
+  - [Run with Docker](#run-with-docker)
+<!-- tocstop -->
+
+## 1. Clone the repository
+
+Using ssh (recommended):
+```bash
+git clone git@github.com:mxenabled/mx-quickconnect.git
+```
+
+Using GibHub CLI:
+```bash
+gh repo clone mxenabled/mx-quickconnect
+```
+
+Using HTTPS:
+```bash
+git clone https://github.com/mxenabled/mx-quickconnect
+```
+
+## 2. Set up development environment
+
+```bash
+cp .env.example .env
+```
+
+Create a .env file at the same level as the `.env.example`.
+Add your `CLIENT_ID` and `API_KEY`.
+You can find these values in your [MX Client Dashboard][]
```

Comment:
Can you add the link to the MX dashboard here? `https://dashboard.mx.com/`

Reply:
It is added at the end of the README file. 

Link is fully useful as you can see here:
<img width="585" alt="image (6)" src="https://user-images.githubusercontent.com/681020/170886982-21b9358b-f23f-4848-8f0e-1ffa825e19f8.png">